docs/tutorials/sinks/1_basic_sink.md

```diff
@@ -258,10 +261,8 @@ emit the event. Change the body of `run_inner` to look like the following:
 +           println!("{}", bytes);
 -           println!("{:#?}", event);
 
-+           emit!(BytesSent {
-+               byte_size: bytes.len(),
-+               protocol: "none".into()
-+           });
++           let bytes_sent = register!(BytesSent::from(Protocol("console".into(),)));
```

Comment:
Small nit - we only need to instantiate this once outside of the while loop and then we can re-use it during every iteration of the loop. Can we change the example to reflect that?